week1/project/hackyourweather/src/main.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import cityData from './city-weather.json';
+
+function MainWeather() {
+  return (
+    <div>
+      {cityData.map((data) => {
+        return (
+          <div className="box" key={data.id}>
+            <City city={data.name} country={data.sys.country} />
+            <Description
+              main={data.weather[0].main}
+              description={data.weather[0].description}
+            />
+            <Temperature min={data.main.temp_min} max={data.main.temp_max} />
+            <Location lon={data.coord.lon} lat={data.coord.lat} />
+          </div>
+        );
+      })}
+    </div>
+  );
+}
+
+function City(props) {
+  const { city, country } = props;
+  return (
+    <h1>
+      {city}, {country}
+    </h1>
+  );
+}
+
+function Description(props) {
```

Comment:
You can destructure in the constructor 
```suggestion
function Description({ main, description }) {
```

Reply:
That is really cool